tasks/init.json

```diff
@@ -10,5 +10,11 @@
       "description": "Run without output ",
       "type": "Optional[Boolean]"
     }
-  }
+  },
+    "implementations": [
```

Comment:
This is using 4 spaces for indentation while the rest of the file use 2 space indentation.

Also, with this change, I experience failures on code I am working on:
```json
{
  "kind": "bolt/run-failure",
  "msg": "Plan aborted: run_task 'yum' failed on 2 nodes",
  "details": {
    "action": "run_task",
    "object": "yum",
    "result_set": [
      {
        "node": "aaa",
        "target": "aaa",
        "action": null,
        "object": null,
        "status": "failure",
        "result": {
          "_error": {
            "kind": "bolt/no-implementation",
            "msg": "No suitable implementation of yum for aaa",
            "details": {
            }
          }
        }
      },
      {
        "node": "bbb",
        "target": "bbb",
        "action": null,
        "object": null,
        "status": "failure",
        "result": {
          "_error": {
            "kind": "bolt/no-implementation",
            "msg": "No suitable implementation of yum for bbb",
            "details": {
            }
          }
        }
      }
    ]
  }
}
```

Reverting this change (or using the latest tagged version of the module, 4.0.0), fixes this issue:

```json
[
  [
    {
      "node": "aaa",
      "target": "aaa",
      "action": "task",
      "object": "yum",
      "status": "success",
      "result": {
        "status": [
          {
            "package": "puppet-agent.x86_64",
            "available_version": "6.6.0-1.el7",
            "repository": "puppet6"
          }
        ]
      }
    },
    {
      "node": "bbb",
      "target": "bbb",
      "action": "task",
      "object": "yum",
      "status": "success",
      "result": {
        "status": [

        ]
      }
    }
  ]
]
```

Reply:
(This is CentOS installed with the AIO package, so I guess something is broken)